Jenkinsfile

```diff
@@ -529,11 +513,7 @@ allESVersions.each { esVersion ->
   quickMatrix.add([task: "test-suite-custom-esversion-force-polyfills", scala: mainScalaVersion, java: mainJavaVersion, esVersion: esVersion, testSuite: "testSuite"])
 }
 allJavaVersions.each { javaVersion ->
-  if (!isExcludedForScala211(javaVersion)) {
-    // the sbt plugin tests want to compile everything for 2.11, 2.12 and 2.13
-    quickMatrix.add([task: "tools-sbtplugin", scala: "2.12.17", java: javaVersion])
-    quickMatrix.add([task: "tools", scala: "2.11.12", java: javaVersion])
-  }
+  quickMatrix.add([task: "tools-sbtplugin", scala: "2.12.17", java: javaVersion])
```

Comment:
It seems we must still exclude this task on JDK 16. The job "tools-sbtplugin scala=2.12.17 java=16" keeps failing.